data-serving/data-service/src/model/case.ts

```diff
@@ -210,14 +236,15 @@ interface Demographics {
 interface ChronicDisease {
     provided: [string];
     imputed: [string];
-    other: [string];
 }
 
-interface EventSequence {
-    onsetSymptoms: Date;
-    confirmed: Date;
-    hospitalAdmission: Date;
-    deathOrDischarge: Date;
+interface EventDate {
```

Comment:
I'm sure I'm missing something obvious -- why define an interface for this?

Reply:
I assumed we'd want to use classes where possible, and not objects (when we actually start doing things with models we read in.)

Otherwise it would be:

interface Event {
    name: string;
    date: {
      range: EventDate;
   }
}

Maybe that's OK, but seems a little weird.

The alternative is to not have a nested object with only 'range' in it. I've been going back and forth on this and would love your input. I've included it in an overabundance of caution in case we ever wanted to go back and add other properties to Event.date (other than range... like if we ever want to put a singular date back in). Maybe that's silly?

The alternative is to just have a compressed Event.dateRange of type Range<Date>. WDYT?

Reply:
I think it's a little too cautious for now. This is one of the reasons we selected a document DB -- embrace the YAGNI! I had your stated alternative in mind.

Reply:
OK, done. 

We'll need to update the mongodb example case json, the mongodb schema json, and the python converter in another PR to reflect this change